symplyphysics/laws/electricity/energy_of_an_electron_in_a_hydrogen_atom.py

```diff
@@ -0,0 +1,31 @@
+from sympy.physics.units import elementary_charge
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+## There is an expression for the total energy of the hydrogen atom according to Bohr's theory.
+
+# Law: E = k * e^2 / (2 * r), where
+## E - total energy of electron,
+## k - Coulomb's constant,
+## e - charge of electron,
+## r - radius of the Bohr orbit of electron.
+
+energy_of_electron = Symbol("energy_of_electron", units.energy)
+
+radius_of_electron = Symbol("radius_of_electron", units.length)
+
+law = Eq(energy_of_electron, units.coulomb_constant * elementary_charge**2 / (2 * radius_of_electron))
```

Comment:
You can refactor other laws and put elementary charge instead of hardcoded value.